acceptance-test/user-pathways/upload-file/upload-file.test.js

```diff
@@ -100,84 +265,56 @@ describe('Upload File(s)', () => {
-            await page.waitForSelector(LOCATION_ENGLAND_OPTION);
-            await page.click(LOCATION_ENGLAND_OPTION);
-            await clickContinueButton(1);
+            await whereAreYouMakingThisReportPage();
 
             // pv-under-age
-            await page.waitForSelector(PV_UNDER_AGE_NO_OPTION);
-            await page.click(PV_UNDER_AGE_NO_OPTION);
-            await clickContinueButton(1);
+            await isThePotentialVictimUnder18Page();
 
             // pv-under-age-at-time-of-exploitation
-            await page.waitForSelector(PV_UNDER_AGE_AT_TIME_OF_EXPLOITATION_NO_OPTION);
-            await page.click(PV_UNDER_AGE_AT_TIME_OF_EXPLOITATION_NO_OPTION);
-            await clickContinueButton(1);
+            await wereTheyUnder18AtAnyTimeDuringTheExploitationPage();
 
             // what-happened
-            await page.waitForSelector(WHAT_HAPPENED_INPUT);
-            await page.$eval(WHAT_HAPPENED_INPUT, (element) => {
-                element.value = 'Test input regarding details of exploitation';
-            });
-            await clickContinueButton(1);
+            await whatDidTheySayHappenedPage();
 
             // where-exploitation-happened
-            await page.waitForSelector(EXPLOITED_IN_UK_OPTION);
-            await page.click(EXPLOITED_IN_UK_OPTION);
-            await clickContinueButton(1);
+            await whereWereTheyLastExploitedPage();
 
             // current-pv-location
-            await page.waitForSelector(CURRENT_PV_LOCATION_UK_REGION);
-            await page.$eval(CURRENT_PV_LOCATION_UK_REGION, (element) => {
-                element.value = 'Rutland';
-            });
-            await clickContinueButton(1);
+            await whereAreTheyNowPage();
 
             // who-exploited-pv
-            await page.waitForSelector(WHO_EXPLOITED_PV);
-            await page.$eval(WHO_EXPLOITED_PV, (element) => {
-                element.value = 'Test details about exploiter(s)';
-            });
-            await clickContinueButton(1);
+            await whoExploitedThemPage();
 
             // types-of-exploitation
-            await clickContinueButton(1);
+            await howWereTheyExploitedPage();
 
             // any-other-pvs
-            await page.waitForSelector(ANY_OTHER_PVS_NO_OPTION);
-            await page.click(ANY_OTHER_PVS_NO_OPTION);
-            await clickContinueButton(1);
+            await wereThereAnyOtherPotentialVictimsPage();
 
             // reported-to-police
-            await page.waitForSelector(PV_HAS_CRIME_REFERENCE_NUMBER_YES_OPTION);
-            await page.click(PV_HAS_CRIME_REFERENCE_NUMBER_YES_OPTION);
-            await clickContinueButton(1);
+            await doTheyHaveACrimeReferenceNumberPage();
 
-            // Run through the skeleton until we reach the upload page
-            await clickContinueButton(13);
+            // pv-want-to-submit-nrm
+            await doTheyWantTheirCaseReferredToTheNrmPage();
```

Comment:
Can you put some of these all in one function and just pass in selector and value